bundle/parameters.go

```diff
@@ -1,14 +1,10 @@
 package bundle
 
-type ParametersDefinition struct {
-	Fields   map[string]ParameterDefinition `json:"fields" mapstructure:"fields"`
-	Required []string                       `json:"required,omitempty" mapstructure:"required,omitempty"`
-}
-
-// ParameterDefinition defines a single parameter for a CNAB bundle
-type ParameterDefinition struct {
+// Parameter defines a single parameter for a CNAB bundle
+type Parameter struct {
 	Definition  string    `json:"definition" mapstructure:"definition"`
 	ApplyTo     []string  `json:"applyTo,omitempty" mapstructure:"applyTo,omitempty"`
 	Description string    `json:"description,omitempty" mapstructure:"description"`
 	Destination *Location `json:"destination,omitemtpty" mapstructure:"destination"`
+	Required    bool
```

Comment:
Looks like this needs a corresponding json identifier still: `json:"required" mapstructure:"required"`